tests/test-timber-menu.php

```diff
@@ -17,7 +20,10 @@ function testBlankMenu() {
 		$this->assertEquals( '/home/', $item->path() );
 	}
 
+<<<<<<< HEAD
+=======
 	
+>>>>>>> 911a9aa41d6fd06942f4f5afff002d6ef184f64f
```

Comment:
Oops. Need to remove the merge lines.

Reply:
Thanks. I'll fix that in a few moments. Not sure how much that'll help the overall checks, though. I really don't get how i missed it either.